juniper-jti/README.md

```diff
@@ -0,0 +1,76 @@
+# Synse Juniper JTI Plugin
+
+The [Synse Juniper JTI Plugin](https://github.com/vapor-ware/synse-juniper-jti-plugin) is a Synse plugin
+which provides networking data from Juniper JTI data streams. 
+
+## TL;DR;
+
+```bash
+$ helm install synse/juniper-jti
+```
+
+## Introduction
+
+This chart bootstraps a [Synse Juniper JTI Plugin](https://github.com/vapor-ware/synse-juniper-jti-plugin)
+deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release synse/juniper-jti
+```
+
+The command deploys a Synse Juniper JTI Plugin on the Kubernetes cluster in the default configuration. The
+[configuration](#configuration) section lists the parameters that can be configured during installation.
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the Synse Juniper JTI Plugin chart and their default values.
+
+| Parameter | Description | Default |
+| :-------- | :---------- | :------ |
+| `config` | The Juniper JTI Plugin configuration. | `{}` |
+| `nameOverride` | Partially override the fullname template (will maintain the release name). | `""` |
+| `fullnameOverride` | Fully override the fullname template. | `""` |
+| `metrics.enabled` | Enable/disable application metrics export (via Prometheus) at `:2112/metrics`. | `false` |
```

Comment:
Nit: If we're going to expose metrics, we should also ship a serviceMonitor template with the application. not something that has to be done now, but worth noting in the review.

Reply:
Is there an example somewhere of a serviceMonitor template I could use as a guide for adding one  here?

Reply:
https://github.com/vapor-ware/cloud-infra/wiki/workload-monitoring

Reply:
question:  since this is a public synse chart, would we want to include this service  monitor  template here, as it feels like  its pretty  specific to how Vapor deploys prometheus on clusters?

Reply:
The only vapor specific thing is the labels we use. ServiceMonitors are part of the PrometheusOperator stack and can be found in various public charts.

eg: https://github.com/helm/charts/blob/728a368f7d6609cdf573b47c2d006e480b9ca534/stable/metallb/templates/servicemonitor.yaml

That particular document was engineered to make it nearly friction-less to include in our charts ;)

Reply:
ah, okay  cool.  just wanted to  make sure  (:

Reply:
added a ServiceMonitor template in the latest commit, ready for review +  comments